### ISSUE TITLE
Infinite restarts of "Starting child process"

### ISSUE THREAD

**Issue by danchistyakov** [petruisfan/node-supervisor#227]
When I just trying to run a very simple code with Supervisor i get infinite restarts of my code.
![Image of logs](https://d.radikal.ru/d01/2109/06/681f0f66a12f.jpg)
Code:
`const Test = () => {
  console.log("test");
};
Test();`
Node.js: v14.17.3;
npm: 7.20.5

**Comment by GusGF** [comment]
Yep I've got the same problem for some reason it doesn't like my code. Ironically exact same script works fine with nodemon which I normally have problems with hence why I generally use Supervisor.

**Comment by userAugustos** [comment]
have u solve this?
I Think it can be a problem with this node version
